clientv3/op.go

```diff
@@ -225,3 +228,11 @@ func WithLastRev() []OpOption { return withTop(SortByModifiedRev, SortDescend) }
 func withTop(target SortTarget, order SortOrder) []OpOption {
 	return []OpOption{WithPrefix(), WithSort(target, order), WithLimit(1)}
 }
+
+// WithProgressNotify makes watch server send periodic status updates.
```

Comment:
progress updates. also please doc that a progress updates has zero events in the response.